website/docs/getting-started/setup/installation-guides/docker/README.md

```diff
@@ -5,172 +5,157 @@ sidebar_position: 1
+Docker is an open-source [containerization](https://www.ibm.com/in-en/cloud/learn/containerization) platform. It enables developers to package applications into standardized executable components called containers. These containers combine the app's source code with the operating system (OS) libraries and dependencies required to run that code in any environment.
 
 <object data="https://www.youtube.com/embed/Tde7GqE6FQQ?autoplay=0" width='750px' height='400px'></object> 
 
 ## Prerequisites
 
+* [Docker-Compose](https://docs.docker.com/compose/install/) (version 1.29.2 or later) _(Recommended)_
 * [Docker](https://docs.docker.com/get-docker/) (version 20.10.7 or later)
-* [Docker-Compose ](https://docs.docker.com/compose/install/)(version 1.29.2 or later)
 
-Create an installation folder called `appsmith` where you would like your Appsmith installation and data storage.
+1. Create an installation folder called `appsmith` where you would like your Appsmith installation and data storage.
+2. `cd` into the installation folder.
 
-`cd` into the installation folder.
-
-## Quick Start (with docker-compose)
+:::tip
+If you'd like to name your installation directory something other than `appsmith`, update your docker-compose.yml file with the name you'd like:
+```yml
+# docker-compose.yml
 
-:::info
-Appsmith [auto-update](./#enabling-appsmith-auto-updates-with-docker-compose) now also supports auto-backup feature. This backup can be used to roll back an update to a previous version of Appsmith, if the user wants to.
+# ...
+services:
+  appsmith:
+    image: index.docker.io/appsmith/appsmith-ce
+    container_name: <YOUR_DIRECTORY_NAME>
+# ...
+```
 :::
 
-The Appsmith Docker image is built with all the components required to run within a single Docker container. All these multiple processes are managed by a Supervisord instance, which is a lightweight process manager.
-
-:::note
-You can expect one container running when using the `docker-compose ps` command. Something like:
+## Setup with docker-compose
 
-_#Container appsmith Running_
-:::
+The Appsmith Docker image contains all the components required to run within a single Docker container. All these multiple processes are managed by a [Supervisord](http://supervisord.org/) instance, which is a lightweight process manager.
 
 ### Docker-compose configuration
 
-:::info
-Currently, auto-update is disabled on the docker-compose file. If you want to enable auto-update for Appsmith, please uncomment all the commented lines in the docker-compose file.
-:::
+Click the link below to download the `docker-compose.yml` file, and place it into your Appsmith installation folder.
```

Comment:
I'm fine with either way -- Tagging @jnikhila since I remember speaking with her about the file icon vs hyperlink also.

Should we make a ticket or something to work on creating/finding a more prominent icon? We don't have a lot of file downloads, but I do agree that the icon doesn't stand out and is easy to miss.

I'll use link text for now.

Reply:
@spark-c / @danciaclara - there is already a CSS class that we are using for this file download icon. We can modify that to make it more prominent. I still feel putting a link is confusing as it looks like an anchor, and having a file download icon will be more helpful.

Reply:
I feel we should go with a link instead of icon because it could be missed if it is separate from the text. especially the case that it is today (users don't know that they have to click that icon to first download the script)
<img width="841" alt="Screenshot 2022-11-23 at 18 43 07" src="https://user-images.githubusercontent.com/32227135/203555493-ed66d7fb-cbde-4836-acb4-34a036919bac.png">

Reply:
@jnikhila @danciaclara  I think I'm also leaning towards the text, particularly because the link text is:

> Click to [download the `docker-compose.yml` file](/img/docker-compose_(3).yml)

Since there is the "Click" instruction along with the highlighted "download \<scriptname\>", I think the intentions should be reasonably clear to the user.

I don't feel super strongly either way and I'm more than happy to go with whichever method; maybe we could loop in a web UX person for their opinion? I feel bad that this PR is already so old though, I don't want to hold it up too much.

Reply:
> I'm fine with either way -- Tagging @jnikhila since I remember speaking with her about the file icon vs hyperlink also.
> 
> 
> 
> Should we make a ticket or something to work on creating/finding a more prominent icon? We don't have a lot of file downloads, but I do agree that the icon doesn't stand out and is easy to miss.
> 
> 
> 
> I'll use link text for now.

Yes, let's go ahead with text. We can revisit if it's not obvious for users and if issue is raised on the doc.